crates/compiler/solve/src/solve.rs

```diff
@@ -1555,6 +1556,17 @@ fn solve(
     state
 }
 
+fn chase_alias_content(subs: &Subs, mut var: Variable) -> (Variable, &Content) {
+    loop {
+        match subs.get_content_without_compacting(var) {
+            Content::Alias(_, _, real_var, _) => {
+                var = *real_var;
+            }
+            content => return (var, content),
```

Comment:
Why is this returning the var as well as mutating it? The caller drops the returned one.